apps/e2e/indicators-storybook-e2e/src/e2e/text-highlight.component.cy.ts

```diff
@@ -0,0 +1,27 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('indicators-storybook', () => {
```

Comment:
We have been manually setting this to include the name of the component - so `indicators-storybook - text-highlight`